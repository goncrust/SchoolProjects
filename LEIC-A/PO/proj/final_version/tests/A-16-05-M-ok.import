CLIENT|cli001|Manuel Pinheiro|103443
CLIENT|cli002|Pedro Pinheiro|103447
CLIENT|cli003|Pedro Oliveira|103449
CLIENT|cli201|Ludgero Oliveira|103440
BASIC|969001|cli001|ON
BASIC|969003|cli001|OFF
BASIC|969006|cli002|ON
BASIC|969007|cli003|ON
BASIC|969008|cli003|SILENCE
FANCY|969011|cli001|ON
FANCY|969013|cli001|SILENCE
FANCY|969014|cli002|ON
FANCY|969015|cli003|ON
FRIENDS|969001|969006,969007,969008,969015
FRIENDS|969006|969001,969003,969013,969014,969015
