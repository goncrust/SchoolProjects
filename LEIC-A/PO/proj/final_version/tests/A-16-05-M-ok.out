MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: BASIC|969001|cli001|IDLE|0|0|969006,969007,969008,969015
BASIC|969003|cli001|OFF|0|0
BASIC|969006|cli002|IDLE|0|0|969001,969003,969013,969014,969015
BASIC|969007|cli003|IDLE|0|0
BASIC|969008|cli003|SILENCE|0|0
FANCY|969011|cli001|IDLE|0|0
FANCY|969013|cli001|SILENCE|0|0
FANCY|969014|cli002|IDLE|0|0
FANCY|969015|cli003|IDLE|0|0
GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: 